src/index.rs

```diff
@@ -94,6 +95,37 @@ impl IndexWriter {
     fn commit_opstamp(&self) -> u64 {
         self.inner_index_writer.commit_opstamp()
     }
+
+    /// Delete all documents containing a given term.
+    fn delete_term(
+        &mut self,
+        field_name: &str,
+        field_value: &PyAny,
+    ) -> PyResult<u64> {
+        if let Some(field) = self.schema.get_field(field_name) {
+            let value = extract_value(field_value).unwrap();
```

Comment:
This unwrap needs to be gone. You can just propagate the error using the `?` operator. Don't forget to document that the method can raise exceptions.